src/main/java/com/google/api/codegen/transformer/py/PythonSurfaceNamer.java

```diff
@@ -130,6 +137,55 @@ public String getGrpcClientTypeName(Interface service) {
   }
 
   @Override
+  public List<String> getThrowsDocLines(MethodConfig methodConfig) {
+    ImmutableList.Builder<String> lines = ImmutableList.builder();
+    lines.add(":exc:`google.gax.errors.GaxError` if the RPC is aborted.");
+    if (hasParams(methodConfig)) {
+      lines.add(":exc:`ValueError` if the parameters are invalid.");
+    }
+    return lines.build();
+  }
+
+  private boolean hasParams(MethodConfig methodConfig) {
+    if (!Iterables.isEmpty(methodConfig.getRequiredFieldConfigs())) {
+      return true;
+    }
+
+    int optionalParamCount = Iterables.size(methodConfig.getOptionalFieldConfigs());
+    return optionalParamCount > (methodConfig.getPageStreaming() == null ? 0 : 1);
```

Comment:
nit: Hard to discern intent from the code here. Leave a comment that this is discounting the `page_size` parameter? (At least I assume that's what this is doing?)